domains/danix.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "danix421",
+        "email": "danix@disroot.org"
+    },
+    "record": {
+        "CNAME": "danix421.github.io"
```

Comment:
404 CNAME record.

Reply:
it's meant to be like that. I haven't commited the page itself yet, since I wanted to work around the domain first.

Reply:
@danix421 We do not support domain reservation, so you will need to at least put something there like maybe some basic contact details and info before you can get the domain.